std::exception: More than one symbol with the name Existing. In %s and %s
